doc/rofi.1.markdown

```diff
@@ -627,35 +619,35 @@ Disable parsing of configuration. This runs rofi in *stock* mode.
 
 Disables the loading of plugins.
 
-To get a trace with (lots of) debug information set the following environment variable when executing **rofi**:
+To get a trace with (lots of) debug information, set the following environment variable when executing **rofi**:
 
 ```
 G_MESSAGES_DEBUG=all
 ```
 
-The trace can be filtered by only outputting the relevant domains, f.e.:
+The trace can be filtered by only outputting the relevant domains, for example:
 
 ```
 G_MESSAGES_DEBUG=Dialogs.DRun
 ```
 
-For more information on debugging see the [wiki](https://github.com/DaveDavenport/rofi/wiki/Debugging%20Rofi)
+For more information on debugging, see the [wiki](https://github.com/DaveDavenport/rofi/wiki/Debugging%20Rofi)
 
 ## PATTERN
 
-To launch commands (e.g. when using the ssh launcher) the user can enter the used command-line,
+To launch commands (e.g. when using the ssh launcher), the user can enter the used command-line,
```

Comment:
you can enter the 'commandline' that rofi uses the launch the application ( the '-run-command' setting). Maybe we should leave this out.